addons/membership/membership_view.xml

```diff
@@ -234,7 +235,7 @@
                                 <label for="membership_state"/>
                                 <div>
                                     <field name="membership_state"/>
-                                    <button name="%(action_membership_invoice_view)d" type="action" string="Buy Membership" 
+                                    <button name="%(action_membership_invoice_view)d" type="action" string="Buy Membership"
```

Comment:
This is an editor issue, it trims spaces at endlines. This is a good practice in programming.